storybook/stories/Stories/UI/Navigation.elm

```diff
@@ -0,0 +1,85 @@
+module Stories.UI.Navigation exposing (..)
+
+import Array
+import Browser
+import Helpers.Layout exposing (col)
+import Html exposing (Html)
+import UI.Click as C
+import UI.Icon as I
+import UI.Navigation as N
+import UI.Nudge
+import UI.Tooltip
+
+
+main : Program () () Msg
+main =
+    Browser.element
+        { init = \_ -> ( (), Cmd.none )
+        , view = \_ -> view
+        , update = \_ currentModel -> ( currentModel, Cmd.none )
+        , subscriptions = \_ -> Sub.none
+        }
+
+
+type Msg
+    = UserClicked
+
+
+sampleNavItem : N.NavItem Msg
+sampleNavItem =
+    N.navItem "some" (C.ExternalHref "some")
+
+
+sampleNavItems : Array.Array (N.NavItem Msg)
+sampleNavItems =
+    [ C.ExternalHref "link 1" |> N.navItem "item 1"
+    , C.ExternalHref "link 2" |> N.navItem "item 2"
+    , C.ExternalHref "link 3" |> N.navItem "item 3"
+    , C.ExternalHref "link 4" |> N.navItem "item 4"
+    ]
+        |> Array.fromList
```

Comment:
What do we get from Array over List here?

Reply:
It was to access one element by its index, but I change it to simply use List.